GitHubRepositories.ps1

```diff
@@ -68,6 +68,9 @@ filter New-GitHubRepository
     .PARAMETER NoWiki
         By default, this repository will have a Wiki.  Specify this to disable the Wiki.
 
+    .PARAMETER Discussions
```

Comment:
I think the parameter would read better if it was named `HasDiscussions` (here and in all references).

Reply:
@HowardWolosky should I resolve this comment after my latest change?